src/lib/components/BucketAggregation/BucketAggregationValues.js

```diff
@@ -67,16 +67,15 @@ class BucketAggregationValues extends Component {
     const valuesCmp = buckets.map((bucket) => {
       const isSelected = this._isSelected(
         this.aggName,
-        bucket.key,
+        bucket.key_as_string ? bucket.key_as_string : bucket.key,
```

Comment:
yessss
@ntarocco why did we reject this change in the past, do you remember?

Reply:
Not sure, I can't remember... I would also add the same change to the label below maybe?

```
const key_field = bucket.key_as_string ? bucket.key_as_string : bucket.key
const label = bucket.label ? bucket.label : `${bucket["key_field"]} (${bucket.doc_count})`
```

Reply:
@ntarocco @kprzerwa I had already an issue with this https://github.com/inveniosoftware/react-searchkit/issues/56 :)